package.json

```diff
@@ -94,5 +93,16 @@
     "@storybook/types": ">=7.5.3",
     "react": ">=16.9.0",
     "react-dom": ">=16.9.0"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "bundler": {
+    "exportEntries": [
+      "src/index.ts"
+    ],
+    "managerEntries": [
+      "src/manager.ts"
+    ]
```

Comment:
Can it be defined in rollup config?

Reply:
Can be, as strings, but here also I went with Storybook's convention
https://github.com/storybookjs/addon-kit/commit/c4f81d7b0ea0faa6f2bb94d22b8835155dee9ac3#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R73